images/capi/ansible/windows/roles/kubernetes/templates/StartKubelet.ps1

```diff
@@ -14,15 +14,15 @@
 
 # From https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/scripts/PrepareNode.ps1
 $FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
-$global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")
```

Comment:
this change is different from the most up-to-date Azure iamge changes so will need to make sure it doesn't break Azure